src/webgl/3d_primitives.js

```diff
@@ -1532,4 +1532,54 @@ p5.RendererGL.prototype.curveVertex = function() {
   }
 };
 
+p5.RendererGL.prototype.image = function(
+  img,
+  sx,
+  sy,
+  sWidth,
+  sHeight,
+  dx,
+  dy,
+  dWidth,
+  dHeight
+) {
+  this._pInst.push();
+
+  this._pInst.texture(img);
+  this._pInst.textureMode(constants.NORMAL);
+
+  // (dx, dy) maps to center of canvas, so it is transformed to the actual (0,0) of canvas
+  var new_dx = dx - this._pInst.width / 2;
```

Comment:
I don't know about this decision. I understand why you would do it. It may be preferable in some cases to have `image()` draw in 2D and 3D in the same way but I think this is a little misleading for new users. The coordinate system for everything else that is drawn in 3D has an origin at screen center. I think this should stay consistent in 3D rather than staying consistent for this particular function. The user would likely expect the top-left corner of the image to be in the center of the sphere in a sketch like this:
```
function draw() {
  background(255);
  push();
  translate(50, 0, 0);
  sphere(50);
  pop();
  image(grid, 50, 0, 200, 200);
}
```
This is definitely open for debate but this is my instinct on this one.